fhir-persistence-jdbc/src/main/java/com/ibm/fhir/persistence/jdbc/dao/impl/ResourceDAOImpl.java

```diff
@@ -200,7 +210,7 @@ public Resource read(String logicalId, String resourceType)
 
     @Override
     public Resource versionRead(String logicalId, String resourceType, int versionId)
-            throws FHIRPersistenceDataAccessException, FHIRPersistenceDBConnectException {
```

Comment:
bad formatting change.  the double-indent signifies that its a continuation of the previous line so that it doesn't look like the first line of the method